h2o-core/src/main/java/water/api/RegisterV3Api.java

```diff
@@ -464,6 +464,14 @@ public void registerEndPoints(RestApiContext context) {
     context.registerEndpoint("list_rest_capabilities",
             "GET /3/Capabilities/API", CapabilitiesHandler.class, "listRest",
             "List of all registered Rest API capabilities");
+
+    context.registerEndpoint("import_grid_search",
```

Comment:
import_grid_search or import_grid?

Reply:
import_grid definíitely

Reply:
Also did the change for export_grid. Done.